posthog/templates/head.html

```diff
@@ -28,7 +28,7 @@
     <script>window.SENTRY_DSN = '{{sentry_dsn}}';</script>
 {% endif %}
 <script id='posthog-app-user-preload'>
-    window.POSTHOG_APP_CONTEXT = {{ posthog_app_context | safe }};
+    window.POSTHOG_APP_CONTEXT = JSON.parse("{{ posthog_app_context | escapejs }}");
```

Comment:
What kinds of problems does this solve? 🤔 

As far as I can tell, `posthog_app_context` is a json dump: https://github.com/PostHog/posthog/blob/master/posthog/utils.py#L274